helper_for_test.go

```diff
@@ -0,0 +1,100 @@
+package mesosAuthPlugin
+
+import (
+	"context"
+	"testing"
+
+	log "github.com/hashicorp/go-hclog"
```

Comment:
Should this be in our dependencies? Does it work because it's a dependency of vault?

Reply:
It's a dependency of vault, and we only need it to set up our test machinery -- we never reference it from non-test code.